Solution1/Solution1.Tests/ConsoleApp/Services/UserCommunicateServiceTests.cs

```diff
@@ -211,13 +212,13 @@ public async Task CommunicateAsync_GetBestWeatherByArrayCityNameAsync_ShowBestWe
             {
                 { true, new List<WeatherResponseDTO>
                             {
-                                new WeatherResponseDTO() { CityName = cityName, IsSuccessfulRequest = true, Temp = temp, LeadTime = leadTime },
-                                new WeatherResponseDTO() { CityName = cityName2, IsSuccessfulRequest = true, Temp = temp2, LeadTime = leadTime2 }
+                                new WeatherResponseDTO() { CityName = cityName, RequestStatus = RequestStatus.Successful, Temp = temp, LeadTime = leadTime },
```

Comment:
there should be tests for cancellation as well

Reply:
+